README.md

```diff
@@ -45,3 +45,20 @@ Check out the [Jekyll docs][jekyll-docs] for more info on how to get the most ou
 [jekyll-docs]: https://jekyllrb.com/docs/home
 [jekyll-gh]:   https://github.com/jekyll/jekyll
 [jekyll-talk]: https://talk.jekyllrb.com/
+
+# Test the blog
+
+To test the blog posts, run:
+
+`make check`
+
+If `make check` does not work, `make generate` might be able to fix things if ran locally, committed and pushed.
```

Comment:
nit: `make check` currently fails when running against Dafny 4.0.0 since some error messages seem to have changed. Not a blocker.